Src/BootCamp.Chapter/Computer/DesktopComputer.cs

```diff
@@ -1,41 +1,28 @@
 ﻿namespace BootCamp.Chapter.Computer
 {
-    public class DesktopComputer
+    public abstract class DesktopComputer
     {
-        private Body _body;
-        public Body GetBody()
+        public DesktopComputer()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the visibility of this constructor to 'protected'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5149533)